src/main_controller.ts

```diff
@@ -141,6 +141,9 @@ export class MainController implements vscode.Disposable {
         );
         if (settings.NVIM_APPNAME) {
             process.env.NVIM_APPNAME = settings.NVIM_APPNAME;
+            if (settings.useWsl) {
+                process.env.WSLENV = "NVIM_APPNAME/u";
```

Comment:
Can you add a one-line comment explaining what "/u" means , or link to the docs that explain WSLENV

Reply:
I added a comment to explain the purpose of the `/u` flag and linked to the documentation.